### PR TITLE
add route for QR codes

### PR DESCRIPTION
**Pull request by phillipthelen**
This adds a route, that redirects a special URL to the profiles page for a user. The idea/reason is that we can then change the profiles url to something nicer, while still keeping existing qr codes usable.

### CODE REVIEW

File: website/server/controllers/top-level/qrcodes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+let api = {};
+
+api.redirectProfileQRCode = {
+  method: 'GET',
+  url: '/qr-code/user/:memberid',
+  runCron: false,
+  async handler (req, res) {
+    req.checkParams('memberid', res.t('memberIdRequired')).notEmpty();
```
Comment (paglias) [inline website/server/controllers/top-level/qrcodes.js]: .isUUID()

**Comment by crookedneighbor** [comment]
Are we considering this a private part of the API? If not, it should be documented.

Could we get a test that it redirects?

**Comment by paglias** [comment]
It's outside the API, I'd say it's better to keep it private

File: test/api/v3/integration/qrcodes/GET-qrcodes_user.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-v3-integration.helper';
+import superagent from 'superagent';
+import nconf from 'nconf';
+
+const API_TEST_SERVER_PORT = nconf.get('PORT');
+describe.only('GET /qr-code/user/:memberId', () => {
```
Comment (crookedneighbor) [inline test/api/v3/integration/qrcodes/GET-qrcodes_user.test.js]: `.only`

**Comment by paglias** [comment]
Looks good to me 👍

**Pull request merged**